vispy/testing/_runners.py

```diff
@@ -190,8 +190,14 @@ def _check_line_endings():
 """
 
 
-def _examples():
-    """Run all examples and make sure they work
+def _examples(individial_file_paths):
+    """Run examples and make sure they work.
+
+    Parameters
+    ----------
+
+    individial_file_paths: str
```

Comment:
maybe this is a Rust thing, but I personally prefer long, descriptive names over short ones.
But very well, I'll change that.

And, I hadn't know the ```paths : str``` style. Will change!